docs/quick_start.md

```diff
@@ -51,69 +81,45 @@ git commit -m "Commit message goes here!"
 
 *Checkout the [Version Control](https://brown-ccv.github.io/honeycomb-docs/docs/version_control) page for more information about working with git*
 
-### 3. Install Electron dependencies
 
-Honeycomb relies on Electron to package the cross-platform desktop applications. Before starting, you will need to install Electron's pre-requisites below. You can find the full instructions on the [electron documentation](https://www.electronjs.org/docs/development/build-instructions-gn) for your specific OS.
 
-#### Windows
+### 3. Install NPM Packages
+With Node.js installed in Step 0, we are now able to utilize the `npm` command in the terminal. 
 
-- Visual Studio:
-Install the latest version of [Visual Studio](https://visualstudio.microsoft.com/downloads/) with the Desktop Development for C++ Workflow. 
-To add the workflow, follow [these instructions](https://docs.microsoft.com/en-us/cpp/build/vscpp-step-0-installation?view=msvc-160#:~:text=If%20you%20have%20Visual%20Studio,Then%2C%20choose%20Modify).
+npm is a command-line utility for Node.js that makes it easy for you to install packages and maintain them throughout the lifecycle of your application. 
 
-- Node.js:
-Install [Node.js](https://nodejs.org/en/download/)
+Honeycomb, like most applications, contains a template called `package.json`. This file contains three distinctive parts that we interact with using a series of npm commands.
```

Comment:
Can these be combined into one paragraph instead of being on separate lines?

Reply:
Definitely. How i see it, we can combine the first two lines. I plan to put the npm install instruction right after this small paragraph. In my opinion the `Honeycomb, like most applications, contains ....` is the start of the subsection that breaks down the `package.json` so I think that should be the start of its own line. What do you think?